src/main/docs/guide/httpClient/httpClientImplementations/javanetClient.adoc

```diff
@@ -0,0 +1,15 @@
+To use an implementation based on https://openjdk.org/groups/net/httpclient/intro.html[Java HTTP Client], add the following dependency to your build:
+
+dependency:micronaut-http-client-javanet[]
+
+NOTE: This implementation of the Micronaut HTTP Client is available since Micronaut Framework 4.0.
+
+The implementation based on https://openjdk.org/groups/net/httpclient/intro.html[Java HTTP Client] does not support the following features:
+
+* Proxy Support.
```

Comment:
We support using an HTTP proxy, but not client.proxy calls...

Reply:
Clarify it.